awscli/paramfile.py

```diff
@@ -37,6 +37,10 @@
     'cloudformation.update-stack.stack-policy-url',
     'cloudformation.set-stack-policy.stack-policy-url',
 
+    'custom.cp.website-redirect',
```

Comment:
Sorry for all the spamming. Actually a change will need to take place at ``unpack_argument`` in ``argprocess.py``: https://github.com/aws/aws-cli/blob/develop/awscli/argprocess.py#L63

The reason being that the service_name/operation_name input limits all commands to the strict two command length structure.

For example, for the custom top layer ``s3`` command, a ``custom`` had to be added to fit the input because both ``service_name`` and ``operation_name`` are required. This is an issue because if I wanted to apply a handler to the event ``custom.s3.*.some-argument`` I would not be able to register to all of ``s3``'s operation commands because ``s3``'s operation commands must emit ``s3.operation.some-argument`` or ``custom.operation.some-argument`` to fit the inputs to that function.

This is also an issue for commands that have more than two commands like waiters.

I propose to leave the PR as is and make this change in a subsequent PR because it is outside the scope of the current bug fix as I will have to change the clidriver and some handlers as well.

@jamesls Let me know what you think.